e2e/e2e_test.go

```diff
@@ -41,6 +41,9 @@ func TestE2e(t *testing.T) {
 	vmConfigFile := filepath.Join(wd, "./../_output/lima-template/fedora.yaml")
 	vmName := "fedora"
 	limaOpt, err := option.New([]string{subject})
+	if err != nil {
+		t.Fatalf("failed to initialize a testing option %v", err)
+	}
```

Comment:
Not needed but I couldn't resist the linter yelling at me. It's VSCodes fault really.